agent/agent.go

```diff
@@ -274,15 +322,17 @@ func (a *Agent) Start() error {
 		}
 
 		a.delegate = server
-		a.state.delegate = server
+		a.State.SetDelegate(server)
+		a.sync.ClusterSize = func() int { return len(server.LANMembers()) }
```

Comment:
How about we do this once after this using the delegate? `a.sync.ClusterSize = func() int ( return len(a.delegate.LANMembers()) }`

Reply:
yes

Reply:
done.